docs/hardware/raspberry-pi/cardboard-control-panel/wires.md

```diff
@@ -2,77 +2,78 @@
+* Repeat this process for **ALL** of the 7 arcade buttons.
 
 ![All buttons wired and ready](/static/hardware/raspberry-pi/cardboard-control-panel/allwires.jpg)
 
-- [ ] grab the 7 **stripped** jumper wires and thread them together. Use electrical tape to keep the cables together.
+* Grab the 7 **stripped** jumper wires and thread them together. Use electrical tape at the end of the unstripped portion to keep the wires together.
 
 ![A bundle of stripped jumper wire cables](/static/hardware/raspberry-pi/cardboard-control-panel/bundle.jpg)
 
-- [ ] strip the **male** connector of a jumper wire and thread it around the bundle of you just created. Use electrical tape to secure it all.
+* Strip the **male** connector of a jumper wire and thread it around the bundle of you just created. Use electrical tape to secure it all.
 
 ![A jumper wire attached to the bundle of jumper wires](/static/hardware/raspberry-pi/cardboard-control-panel/gndwire.jpg)
 
 ### ~ hint
 
-If you have access to a soldering iron, we strongly recommend to add a bit of solder on the threaded cables to guarantee a great electrical connection.
+#### Secure wire joints
+
+If you have access to a soldering iron, it's a good idea to add a bit of solder where the threaded wires join to guarantee a good electrical connection that won't come apart.
 
 ### ~
 
 ## Prepping the joysticks
 
-For each of joysticks,
+For each of joysticks, they have 4 direction switches to prepare:
 
-- [ ] connect the joystick cable on **COM** and **NO** terminals.
+* Connect the joystick cable to the **COM** and **NO** terminals of the switch.
 
 ![Connection wires on the joystick terminals](/static/hardware/raspberry-pi/cardboard-control-panel/joystick.jpg)
 
-- [ ] connect 2 jumper wires to the JST connector.
-- [ ] strip one jumper wire for each connector.
-
-Repeat these steps for the 4 connectors of the joystick.
+* Connect 2 jumper wires to the JST connector.
+* Strip one jumper wire for each connector.
+* Repeat these steps for the terminals on **ALL** 4 of the joystick switches.
 
 ![Joystick cables with jumper wires attached from the JST connectors](/static/hardware/raspberry-pi/cardboard-control-panel/joystickconnectors.jpg)
 
-
 ### ~ hint
 
-You can use small zip ties to bundle cables together and avoid a spaghetti-like mess of wires.
+#### Tidy up the wires
+
+You can use small zip ties to bundle wires together and avoid a spaghetti-like mess of wires.
 
 ![A small zip tie keep wires together](/static/hardware/raspberry-pi/cardboard-control-panel/joystickzip.jpg)
 
 ### ~
 
-- [ ] join the 4 stripped jumper wire into a bundle.
+* Join the 4 stripped jumper wires into a bundle.
 
 ![A bundle of stripped jumper cables from the joystick terminals](/static/hardware/raspberry-pi/cardboard-control-panel/joystickbundle.jpg)
 
-- [ ] strip the **male** connector of a jumper wire and connect it to the bundle.
+* Strip the **male** connector of a jumper wire and connect it to the bundle.
```

Comment:
"cut off" the male connector and twist the bare wire to the bundle?

Reply:
You have to cut and strip the platisc case

Reply:
You have to cut and strip the platisc case